lib/runtime/wasmer/imports.go

```diff
@@ -1187,7 +1197,8 @@ func ext_default_child_storage_storage_kill_version_1(context unsafe.Pointer, ch
 	storage := ctx.Storage
 
 	childStorageKey := asMemorySlice(instanceContext, childStorageKeySpan)
-	storage.DeleteChild(childStorageKey)
+	err := storage.DeleteChild(childStorageKey)
+	panicOnError(err)
```

Comment:
There are a lot of places in our runtime code where we should panic (or somehow handle the error), but I'd say it's out of scope since code was not modified there. Let's address this in #2669 instead